test/spec/features/space-tool/SpaceToolSpec.js

```diff
@@ -637,6 +639,158 @@ describe('features/space-tool', function() {
   });
 
 
+  describe('steps', function() {
```

Comment:
What is the actual scenarios that we fix with this new version of the tool? I presume there are some?

If that is the case, can we create full tests for these scenarios, too? Otherwise we'll not be able to verify what we fixed now is still working after the next rewrite of the space tool :tm: :rabbit: :copyright: .

Reply:
I'll create integration tests in bpmn-js for some edge cases.